libs/order-list/src/lib/components/order-list/order-list.tsx

```diff
@@ -27,20 +81,19 @@ export const OrderList = forwardRef<AgGridReact, OrderListProps>(
         defaultColDef={{ flex: 1, resizable: true }}
         style={{ width: '100%', height: '100%' }}
         getRowId={({ data }) => data.id}
+        rowHeight={40}
       >
         <AgGridColumn
-          headerName={t('Market')}
+          headerName="Market"
           field="market.tradableInstrument.instrument.code"
         />
         <AgGridColumn
-          headerName={t('Amount')}
+          headerName="Amount"
           field="size"
           cellClass="font-mono"
           valueFormatter={({ value, data }: ValueFormatterParams) => {
             const prefix = data.side === Side.Buy ? '+' : '-';
-            return (
-              prefix + addDecimal(value, data.market.positionDecimalPlaces)
```

Comment:
chore (blocking): Why are you removing this? Amount still needs formatting with DPs due to fractional orders